packages/react/src/lib/felaExpandCssShorthandsPlugin.ts

```diff
@@ -0,0 +1,33 @@
+import * as _ from 'lodash'
+import * as _expand from 'css-shorthand-expand'
+
+// `css-shorthand-expand` is a CJS library, there are known issues with them:
+// https://github.com/rollup/rollup/issues/1267#issuecomment-446681320
+const expand = (_expand as any).default || _expand
+
+export default () => {
```

Comment:
NIT: what about defining the method outside instead of using anonymous method?

```typescript
const expandCssShorthands = styles => {
  // function body
}

export default expandCssShorthands
```

reasons:
- more readable
- can leverage automatic import by name

Reply:
This is the definition that the fela renderer expect from a plugin. Other internal plugins that we have are defined in the same manner. Would rather not change at this moment.